### ISSUE TITLE
Add compilation defaults for the MaskedLM task models

### ISSUE THREAD

**Issue by mattdangerw** [keras-team/keras-nlp#830]
In https://github.com/keras-team/keras-nlp/issues/709 we added compilation defaults to all classification tasks. We should also do the same for all `XXMaskedLM` models. E.g. [bert](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/bert/bert_masked_lm.py). Here's the full list of models.

- [ ] `AlbertMaskedLM`
- [ ] `BertMaskedLM`
- [ ] `DebertV3MaskedLM`
- [ ] `DistilBertMaskedLM`
- [ ] `FNetMaskedLM`
- [ ] `RobertaMaskedLM`

https://github.com/keras-team/keras-nlp/pull/714 is a good template PR. We can probably just use Adam and `1e-5` or `2e-5` as a learning rate for now. Though a little experimentation for each model to make sure the task does converge would be helpful. This [colab](https://gist.github.com/mattdangerw/b16c257973762a0b4ab9a34f6a932cc1) may be a helpful starting place.

**Comment by Ujjal99kumar** [comment]
I am new here to contribute open source and looking for issues. Can I try this issue, I have programming knowledge on python.

**Comment by shivance** [comment]
Hi @mattdangerw , I can take this up !

**Comment by shivance** [comment]
@mattdangerw How about this ... @Ujjal99kumar seems a beginner, but this issue is not good first issue. We can let him have crack at `BertMaskedLM` while I will raise PRs for rest of models ! I can review his pr as per review, helping him.

**Comment by shivance** [comment]
Sounds good @Cyber-Machine ! Let's split up, let @Ujjal99kumar try Bert, and you can take Deberta and FNet !
Thanks !

**Comment by TheAthleticCoder** [comment]
I can take up DistilBertMaskedLM or AlbertMaskedLM. Thanks!

**Comment by soma2000-lang** [comment]
@mattdangerw  I ahve opened a pr for FNetMaskedLM

**Comment by ADITYADAS1999** [comment]
hey I take DebertV3MaskedLM

**Comment by daripellimanish** [comment]
can i take part in this and also this is my first contribution
